core/templates/dev/head/pages/exploration_editor/ExplorationDataService.js

```diff
@@ -72,13 +75,9 @@ oppia.factory('ExplorationDataService', [
           // We can safely remove the locally saved draft copy if it was saved
           // to the backend.
           LocalStorageService.removeExplorationDraft(explorationId);
-          if (successCallback) {
-            successCallback(response);
-          }
+          successCallback(response);
         }, function() {
```

Comment:
You could probably just replace these three lines with `errorCallback`.

Reply:
Done.